### ISSUE TITLE
[Question] Movie/TV show information fetched from target information instead of TMDB

### ISSUE THREAD

**Issue by juanma-cvega** [jurialmunkey/plugin.video.themoviedb.helper#176]
I don't know if this question is actually related to TheMovieDB Helper, the skin or Kodi in general. I realised when playing something, the information the OSD pulls up comes from Emby instead of TMDB. This causes that for TV shows where the information was scraped from TVDB, it shows wrong information like season or episode numbers compared to those of TMDB. Is there any way to prefer TMDB information over the addon's? For some other addons like Venom, this causes some missing altogether like plot or score.

**Comment by jurialmunkey** [comment]
TMDbHelper passes all the item details as a listitem to the Player - if the addon being called then resolves the file url and includes its own listitem details, that will override the details provided by TMDbHelper.

Basically I'm doing all that I can from my end but ultimately it depends on how the addon at the other end resolves files. The other addon isn't doing anything incorrect - it's just that there isn't really any sane way to then alter the properties it sets since once Kodi's player function is called, TMDbHelper exits. 

Basically TMDbHelper would have to hang around and monitor the Kodi player and then alter the info after it begins -- which could get messy quickly, particularly if something failed or the wrong file played etc. etc.

**Comment by juanma-cvega** [comment]
Ok, makes sense then. I actually thought it was on Kodi side to, given the title of the video playing, find the details at the last part with information providers. So basically, each addon would need to add the logic to only fill the missing information or nothing at all for this to work. 
I always wondered why addons like Venom have missing information when videos are played through OpenMeta/TheMovieDB helper but everything is there when played directly through the addon. Would that be because there is information not being sent to the addon through the player for it to go find it, like the plot and the scores in this case?

**Comment by jurialmunkey** [comment]
> Would that be because there is information not being sent to the addon through the player for it to go find it, like the plot and the scores in this case?

I don't use that addon, so I wouldn't know - but this is probably the case. Some addons pass metadata through the plugin url, so if you're not sending the same metadata through as the plugin would from the list then you wouldn't get the same info when playing.
